cmd/nerdctl/image_convert.go

```diff
@@ -387,3 +484,34 @@ func imageConvertShellComplete(cmd *cobra.Command, args []string, toComplete str
 	// show image names
 	return shellCompleteImageNames(cmd)
 }
+
+type imageInfo struct {
```

Comment:
This name  will cause confusion when placed in the `main` package.
Should be moved to somewhere under `pkg`